pkg/config/config.go

```diff
@@ -28,6 +30,8 @@ func Get() Config {
 			AllowOrigins:     getEnvSlice("ACCESS_CONTROL_ALLOW_ORIGIN", []string{"*"}, ","),
 			AllowCredentials: getEnvBool("ACCESS_CONTROL_ALLOW_CREDENTIALS", false),
 		},
+		StaticFilesDir: getEnv("STATIC_FILES_DIR", ""),
+		ApiRoutePrefix: "/api/v1", //should start with a "/",
```

Comment:
should split the version into a separate config property that can be read from an environment variable and defaults to v1.

Reply:
Split into 2 config properties now - just confirming, do we want the api version to be able to be set by env var?

Reply:
resolved by your below reply